x11-packages/wlroots/build.sh

```diff
@@ -3,22 +3,24 @@ TERMUX_PKG_DESCRIPTION="Modular wayland compositor library"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
```

Comment:
Since the wlroots will contain significant amounts of patches written by you after this PR, would you mind also setting yourself as the maintainer for the wlroots package?